deploy/template.yaml

```diff
@@ -1087,9 +1059,18 @@ Resources:
       AttributeDefinitions:
         - AttributeName: "ipvSessionId"
           AttributeType: "S"
+        - AttributeName: "authorizationCode"
+          AttributeType: "S"
       KeySchema:
         - AttributeName: "ipvSessionId"
           KeyType: "HASH"
+      GlobalSecondaryIndexes:
+        - IndexName: "authorizationCode-index"
```

Comment:
Do we need to namespace this with `index`. In context is it obvious that this is an index? Does it clash with the attribute name somewhere elsewhere?

Reply:
Doesn't seem to be anything against this in the documentation, but I've tried it and it doesn't see to want to create the index. Also noticed if you try to create an index manually in the console it does add `-index` on the end